enterprise/migrations/0086_auto_20200128_1347.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.27 on 2020-01-28 13:47
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import enterprise.validators
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('enterprise', '0085_enterprisecustomeruser_linked'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='enterprisecustomer',
+            name='contact_email',
+            field=models.EmailField(blank=True, help_text='Email for intended use on learner portal Need Help section.', max_length=254, null=True),
```

Comment:
I'd recommend changing the help text to be more generic and not tied to a specific UI feature. Perhaps, "Email for the Learning Coordinator that learners can reach out to with questions." That way, if/when the use of this field in the learner portal changes, we won't need to update the help text.